tests/config/functional/toml/unknown_msgid/enable_unknown_msgid.2.out

```
@@ -0,0 +1,3 @@
+************* Module {abspath}
```

This kind of silent fail are a pain to deal with, glad it's fixed !